src/Ticket.php

```diff
@@ -1747,8 +1747,8 @@ public function prepareInputForAdd($input)
                 foreach ($items as $items_id) {
                     if ($item = getItemForItemtype($itemtype)) {
                         $item->getFromDB($items_id);
-                        $input['_states_id_of_item']    = $item->fields['states_id'];
-                        $input['_locations_id_of_item'] = $item->fields['locations_id'];
+                        $input['_states_id_of_item']    = isset($item->fields['states_id']) ?? 0;
```

Comment:
This sounds strange to me. This would be enough:
```suggestion
                        $input['_states_id_of_item']    = $item->fields['states_id'] ?? 0;
```